eureka-core/src/main/java/com/netflix/eureka/AbstractInstanceRegistry.java

```diff
@@ -0,0 +1,1282 @@
+/*
```

Comment:
Almost no change, except extraction of interface. Unfortunately git treats this as a new file.